### ISSUE TITLE
Configure SAC to disabled upon install

### ISSUE THREAD

**Issue by JanKrivanek** [dotnet/sdk#38256]
Blocked by: https://microsoft.visualstudio.com/OS/_workitems/edit/48660765

### Context

[Smart App Control](https://support.microsoft.com/en-au/topic/what-is-smart-app-control-285ea03d-fa88-4d56-882e-6698afdb7003) curently performs cloud based monitoring in [evaluation mode](https://support.microsoft.com/en-au/topic/what-is-smart-app-control-285ea03d-fa88-4d56-882e-6698afdb7003#ID0EFL), intercepting I/Os on the machine and is able to detect a developer machine only upon some initial extended period (usually 24 - 72 hours) possibly slowing up an initial experience of users experimenting with new setup.
We saw cases where users decided to revert their upgrades (OS/VS/SDK), thinking they lead to inferior performance (some details in linked ADO item). We are cooperating with Smart App Control team to allow client side check facilitated via registry key (yet to be defined). On the SDK and VS side we shoud take care about setting the key during install, to prevent the initial performance hit.

**Comment by JanKrivanek** [comment]
> Does Smart App Control detect [Developer Mode](https://learn.microsoft.com/windows/apps/get-started/enable-your-device-for-development) quickly?

24 - 72 hours. Which might be longer than some users invest in evaluating new setup

**Comment by JanKrivanek** [comment]
Sorry I misread. Smart App Control currently has it's own way of determining the machine is a development machine (and that detection has the mentioned duration)

**Comment by JanKrivanek** [comment]
That's correct and that's hopefully going to happen mid-/long- term. For short term we're only getting this way around.

**Comment by nagilson** [comment]
A spec was made on the VS Side... still waiting for the spec.
